frontend/src/components/eMIB/Emib.jsx

```diff
@@ -6,52 +6,48 @@ import LOCALIZE from "../../text_resources";
 import mini_banner from "./../../images/mini_banner.png";
 
 const PAGES = {
-  welcome: "welcome",
   howTo: "howTo",
   emibTabs: "emibTabs",
   confirm: "confirm"
 };
 class Emib extends Component {
   state = {
-    curPage: PAGES.welcome
+    curPage: PAGES.howTo
   };
 
   changePage = () => {
     switch (this.state.curPage) {
-      case PAGES.welcome:
-        this.setState({ curPage: PAGES.howTo });
-        break;
       case PAGES.howTo:
         this.setState({ curPage: PAGES.emibTabs });
         break;
       case PAGES.emibTabs:
         this.setState({ curPage: PAGES.confirm });
         break;
       default:
-        this.setState({ curPage: PAGES.welcome });
+        this.setState({ curPage: PAGES.howTo });
         break;
     }
   };
 
   render() {
     return (
       <div className="app">
-        {(this.state.curPage === PAGES.welcome || this.state.curPage === PAGES.howTo) && (
-          <img src={mini_banner} alt="" className="banner" />
+        {this.state.curPage === PAGES.howTo && <img src={mini_banner} alt="" className="banner" />}
+        {this.state.curPage === PAGES.howTo && (
```

Comment:
Is there a way to combine line 35 and line 36, since these are for the same state?

Reply:
yes, one sec

Reply:
resolved in ff473cb